deploy/inventree.conf

```diff
@@ -0,0 +1,39 @@
+; # Supervisor Config File
+; Example configuration file for running InvenTree using supervisor
+; There are two separate processes which must be managed:
+;
+; ## Web Server
+; The InvenTree server must be launched and managed as a process
+; The recommended way to handle the web server is to use gunicorn
+;
+; ## Background Tasks
+; A background task manager processes long-running and periodic tasks
+; InvenTree uses django-q for this purpose
+
+[supervisord]
+; Change this path if log files are stored elsewhere
+logfile=/home/inventree/log/supervisor.log
+nodaemon=true
+
+[program:inventree-server]
+user=inventree
+directory=/home/inventree/src/InvenTree
+command=/home/inventree/env/bin/gunicorn -c gunicorn.conf.py InvenTree.wsgi
+startsecs=10
+autostart=true
+autorestart=true
+startretries=3
+; Change these paths if log files are stored elsewhere
+stderr_logfile=/home/inventree/log/server.err.log
+stdout_logfile=/home/inventree/log/server.out.log
+
+[program:inventree-cluster]
+user=inventree
+directory=/home/inventree/src/InvenTree
+command=/home/inventree/env/bin/python manage.py qcluster
```

Comment:
So this needs to be something like this.

```
command=sh -c 'DJANGO_SETTINGS_FILE=InvenTree.settings /home/inventree/env/bin/python /home/inventree/src/InvenTree/manage.py qcluster'
```
But this fails immediately because config.yaml isn't found.  So generation of an initial config.yaml file needs to be done.